rafs/src/lib.rs

```diff
@@ -224,9 +227,69 @@ impl dyn RafsIoRead {
     }
 }
 
+/// A trait to unpack a Rafs filesystem to a tarball or directory.
+pub trait RafsUnpack {
```

Comment:
Is any place using it?

Reply:
Sorry, I'm refactor `nydus-image pack` and this trait should be introduced later.